src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/MLDsa/MLDsaTestsBase.cs

```diff
@@ -222,53 +222,95 @@ public void NistImportSecretKeyVerifyExportsAndSignature(MLDsaNistTestCase testC
             Assert.Equal(testCase.ShouldPass, mldsa.VerifyData(testCase.Message, testCase.Signature, testCase.Context));
         }
 
-        [Theory]
-        [MemberData(nameof(MLDsaTestsData.IetfMLDsaAlgorithms), MemberType = typeof(MLDsaTestsData))]
-        public void ImportPublicKey_Export(MLDsaKeyInfo info)
+        [Fact]
+        public virtual void ImportPublicKey_ExportPublicKey()
```

Comment:
I don't like the idea of a test getting virtualized.  We want the test to test what we want it to test.

Making "make your assertions about when the key format/material isn't available" a virtual seems fine; but not the test.